### PR TITLE
Misc clang-tidy/linter/styleguide fixes in OrbitGl

### PR DESCRIPTION
**Pull request by dpallotti**
I wanted to rename the last `a_ArgName`s to `arg_name`, got carried away.

### CODE REVIEW

File: src/OrbitGl/Batcher.h [hunk 1]
```diff
@@ -103,9 +103,9 @@ class Batcher {
  public:
   explicit Batcher(BatcherId batcher_id, PickingManager* picking_manager = nullptr)
       : batcher_id_(batcher_id), picking_manager_(picking_manager) {
-    constexpr float kSteps = 22;
+    constexpr int32_t kSteps = 22;
```
Comment (beckerhe) [inline src/OrbitGl/Batcher.h:106]: My favourite! :sweat_smile:

File: src/OrbitGl/TextRenderer.cpp [hunk 2]
```diff
@@ -25,11 +25,11 @@
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/Tracing.h"
 
-typedef struct {
+using vertex_t = struct {
```
Comment (beckerhe) [inline src/OrbitGl/TextRenderer.cpp:28]: What about `struct vertex_t { ... };` as usual?

**Review by beckerhe — APPROVED** [review]
This is very cool. Thank you a lot, Davide!

I added a minor comment. Otherwise I haven't seen anything suspicious. LGTM!

File: src/OrbitGl/TextRenderer.cpp (hunk 2 above)
Comment (dpallotti) [reply src/OrbitGl/TextRenderer.cpp]: Hehe I got distracted by the existing `typedef`. Done.

**Pull request merged**